### ISSUE TITLE
generate bundle with a different targets

### ISSUE THREAD

**Issue by rickiesmooth** [prateekbh/babel-esm-plugin#42]
In my babel config I configured my `preset-env` to target IE11 and higher, and when I added the babel-esm-plugin I thought it would create another bundle which targets `esmodules` regardless of the targets that are defined in the babel config (in my case IE11).

When I do a standalone build that targets esmodules the filesize is considerably smaller and preserves ES6 features like classes etc, but when I use the plugin with my babel config and inspect the ES6 bundle geneerated by this plugin it doesn't preserve ES6 features.

Is this expected behavior or am I doing something wrong? If it's unclear I could setup a test case to provide more information.

**Comment by rickiesmooth** [comment]
Oops, I should have read the docs better but I had my preset-env configured in my `babel.config.js` file and didn't configure it as described in the docs. Fixed that and works as expected 👍

**Comment by ForsakenHarmony** [comment]
That's weird, you do get an additional bundle as the output though?

**Comment by prateekbh** [comment]
Expected behavior  is exactly what you’re  looking for .
we can use some help for a repro of the bug

**Comment by rickiesmooth** [comment]
thanks for your replies! Yes the `.es6` file is getting generated, and while I was setting up my test case, I see that it does work as expected. For some reason in my project it isn't..I'll investigate some more.

**Comment by rickiesmooth** [comment]
After some digging I found the issue in my implementation, in my case I had multiple babel-loaders rules: one that only included application code and excluded `node_modules` and one that only included `node_modules` - which caused this issue with the `babel-esm-plugin`.

My workaround was removing my `node_modules` rule and doing everything with one `babel-loader` rule (which eventually was better anyway) so I just wanted to let you know and see if this is a usecase you want to support?
I really enjoyed reading the blogpost about making this module and tried out some fixes, but I couldn't solve this issue myself. Maybe if you'd give me some pointers (and want to support this use case) I can make a PR.

Here's the test case I made to help clarify the issue:  
* **config**: https://github.com/rickiesmooth/babel-esm-plugin/blob/task/reproducible-example-of-issue-with-multiple-loaders/tests/es5-target/es5-target.test.js
* **actual output**: https://github.com/rickiesmooth/babel-esm-plugin/blob/task/reproducible-example-of-issue-with-multiple-loaders/tests/es5-target/fixtures/output.es6.js
